### PR TITLE
[ChangeLog] Add states to Watch the video button in post

### PR DESCRIPTION
**Pull request by junhouse**
Light
Screenshot 2024-11-08 at 3 41 48 PM

Screenshot 2024-11-08 at 3 41 51 PM

Dark
Screenshot 2024-11-08 at 3 41 35 PM
Screenshot 2024-11-08 at 3 41 38 PM
Screenshot 2024-11-08 at 3 41 43 PM

### CODE REVIEW

File: pages/overrides.scss [hunk 1]
```diff
@@ -1522,6 +1559,35 @@ article img {
       }
     }
 
+    .thumbnailPlayButton {
```
Comment (lindseybradford) [inline pages/overrides.scss:1562]: Worth adding `1564`, `1571`, `1577` to the `colors` API?

**Review by lindseybradford — APPROVED** [review]

**Review by ryancurtis1 — APPROVED** [review]

File: components/VideoButtonWithModal/VideoButtonWithModal.tsx [hunk 2]
```diff
@@ -22,8 +22,17 @@ export default function VideoButtonWithModal({
 
   // TODO: update this style and abstract it as time allows to a single button component
   // https://www.figma.com/design/8kiticjQNChvsP9y7s9SRf/Product-Releases-(Copy)?node-id=982-75355&node-type=frame&t=O7vwnwoAoOx42stw-0
+
+  let playButtonBaseClass =
```
Comment (matthathornmp) [inline components/VideoButtonWithModal/VideoButtonWithModal.tsx]: Would suggest making this a file-level const, e.g. `const PLAY_BUTTON_BASE_CLASS = "nx-flex...."`. Then you can avoid the mutable state:
```Typescript
  const playButtonStyles = showThumbnail ? `thumbnailPlayButton ${PLAY_BUTTON_BASE_CLASS}` : `playButton ${PLAY_BUTTON_BASE_CLASS};
```

File: pages/overrides.scss (hunk 1 above)
Comment (matthathornmp) [reply pages/overrides.scss:1562]: At the very least, `#FFF` is already something we can set to `colors.$white`.

**Review by matthathornmp — COMMENTED** [review]
LGTM with a couple small comments.

File: pages/overrides.scss (hunk 1 above)
Comment (junhouse) [reply pages/overrides.scss:1562]: Updated the FFF, but for the other hex codes for now it's one-off usage of hex codes, so let me talk to designers and see if we should name them. I will follow up

**Pull request merged**